pages/about.html

```diff
@@ -5,8 +5,28 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="stylesheet" href="../stylesheets/style.css">
+  <link rel="stylesheet" href="../stylesheets/about.css">
 </head>
-<body>
-  
+<body id="main-grid">
+  <header id="header">
+  <nav>
+    <ul class="container">
+      <li><a href="index.html" >home.</a></li>
+      <li><a href="portfolio.html" >portfolio.</a></li>
+      <li><a href="about.html" >about.</a></li> 
+    </ul>
+  </nav>
+  </header>
+  <h1>About Me</h1>
+  <section id="blurb">
+    <p> A life-long learner who has met her match in a field that revolves around learning constantly, problem-solving on the fly, and often flying by the seat of your pants.
+      Coming from a background in operations, Tara utilizes data to drive solutions that meet needs while supporting healthy work culture for all. 
+      All that has changed is the language (or, languages) used to reach those solutions. </p>
+    <br>
+    <img src="../images/computer image.jpeg" alt="old-school computer graphic">
```

Comment:
Image URLs should not have a space in them, it causes errors. Make sure when you add assets and name them to remove spaces!